front/src/modules/trainschedule/components/ImportTimetableItem/ImportTimetableItemConfig.tsx

```diff
@@ -237,6 +246,19 @@ const ImportTimetableItemConfig = ({
       });
     });
 
+    // TrainGroups refer to missions
+    const missions: Record<string, ImportedTrainSchedule[]> = {};
```

Comment:
"mission" is a French term. The English term is "service", but we use "paced train" in the codebase.